src/store/models/network.spec.ts

```diff
@@ -1023,6 +1024,52 @@ describe('Network model', () => {
     });
   });
 
+  describe('ManualMineCount', () => {
+    beforeEach(async () => {
+      await store.getActions().network.addNetwork(addNetworkArgs);
+    });
+
+    it('should set manual mine count for a network', () => {
+      const { setManualMineCount } = store.getActions().network;
+      const networkId = firstNetwork().id;
+
+      expect(firstNetwork().manualMineCount).toBe(6);
+      setManualMineCount({ id: networkId, count: 10 });
+      expect(firstNetwork().manualMineCount).toBe(10);
+    });
+
+    it('should fail to set manual mine count with invalid network id', () => {
+      const { setManualMineCount } = store.getActions().network;
+
+      expect(() => setManualMineCount({ id: 999, count: 10 })).toThrow(
+        "Network with the id '999' was not found.",
+      );
+    });
+
+    it('should update manual mine count and persist changes', async () => {
+      const { updateManualMineCount } = store.getActions().network;
+      const networkId = firstNetwork().id;
+
+      (injections.dockerService.saveComposeFile as jest.Mock).mockClear();
+      (injections.dockerService.saveNetworks as jest.Mock).mockClear();
```

Comment:
These lines are unnecessary because we configure jest with `"resetMocks": true` in `package.json. So the mocks are already cleared after every test.